libs/blocks/global-navigation/blocks/profile/profile.css

```diff
@@ -0,0 +1,84 @@
+.feds-profile-menu {
+  position: absolute;
+  width: 320px;
+  right: 0;
+  top: 100%;
+  border: 1px solid #e1e1e1;
+  border-radius: 0 0 4px 4px;
+  background: var(--color-white);
+  overflow: hidden;
+  box-shadow: 0 2px 6px -1px rgb(0 0 0 / 10%);
+  animation: profile-collapsed .2s ease 0s 1 both;
+}
+
+.feds-profile.is-open .feds-profile-menu {
+  display: block;
+  animation: profile-expanded .2s ease 0s 1 both;
+}
+
+.feds-profile-header {
+  padding: 20px;
+  display: flex;
+  flex-direction: row;
+  border-bottom: 1px solid #e1e1e1;
+}
+
+.feds-profile-header .feds-profile-img {
+  height: 64px;
+  width: 64px;
+}
+
+.feds-profile-details {
+  margin-left: 16px;
+  overflow: hidden;
+}
+
+.feds-profile-name {
+  margin: -3px 0 0;
```

Comment:
Some weird margins taken over from the original profile to keep things consistent - not sure if we'd rather just want to drop it though.